README.md

```diff
@@ -1,21 +1,24 @@
 # JQuery/Ajax Quiz
 
-Write your answer below each question. 
+Write your answer below each question.
 
 ### Question 1
-(a) What is a click handler? 
+(a) What is a click handler?
+- a click handler is a method where we can set functionality for user input event, such as a mouse click, mouseover or mouseup action that will execute a function.
 
-(b) Where in your JS file should you put it, and WHY? 
+(b) Where in your JS file should you put it, and WHY?
+- it should be in the document.ready section so that it will be ready for the user to use it.
 
 ### Question 2
-If you get the error `$ is undefined`, what does that mean and what could you do to fix it? 
+If you get the error `$ is undefined`, what does that mean and what could you do to fix it?
+- it means that the variable $ was not defined at the time it was called upon.
```

Comment:
In general, yes. But more specifically, 99% of the time this error means you haven't included JQuery correctly.